### PR TITLE
initial setup for header and footer

### PR DESCRIPTION
**Pull request by crgjr66**
- updated header component and appropriate page links
- updated footer component and appropriate page links
- added some styling
- still needs additional styling to match wireframe

### CODE REVIEW

File: app/javascript/components/components/Footer.js [hunk 1]
```diff
@@ -1,13 +1,30 @@
...
+            sign_out_route,
+        } = this.props
         return (
-            <React.Fragment>
-                <h3>Footer</h3>
-            </React.Fragment>
-        );
+            <>
+                <div id="footer">
+                <Nav>
```
Comment (sjproctor) [inline app/javascript/components/components/Footer.js]: This nested Nav tag should be indented and there is an extra line on 22

File: app/javascript/components/components/Header.js [hunk 2]
```diff
@@ -1,13 +1,50 @@
...
+                        </NavItem>
+                        <NavItem>
+                            <a href="/search">New Search</a>
+                        </NavItem>
+                        {logged_in &&
+                            <>
+                                <NavItem>
+                                    <a href={sign_out_route}>Sign Out</a>
+                                </NavItem>
+                                // <NavItem>
```
Comment (sjproctor) [inline app/javascript/components/components/Header.js]: Commented code should be removed from PRs

**Review by sjproctor — CHANGES_REQUESTED** [review]
Couple small things to fix. It might be worth a conversation among your team to decide on a basic style guideline to keep spacing and indentation consistent.

File: app/javascript/components/components/Footer.js (hunk 1 above)
Comment (crgjr66) [reply app/javascript/components/components/Footer.js]: made the requested changes

File: app/javascript/components/components/Header.js (hunk 2 above)
Comment (crgjr66) [reply app/javascript/components/components/Header.js]: made the requested changes

**Review by sjproctor — APPROVED** [review]
Looks great!

**Pull request merged**